### PR TITLE
ALS-7581: Implement multiple values per patient in CSV output

### PR DESCRIPTION
**Pull request by ramari16**

### CODE REVIEW

File: service/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/service/QueryServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        Query.VariantInfoFilter variantInfoFilter = new Query.VariantInfoFilter();
+        variantInfoFilter.categoryVariantInfoFilters = Map.of("Gene_with_variant", new String[]{"LOC102723996", "LOC101928576"});
+        variantInfoFilters.add(variantInfoFilter);
+        query.setVariantInfoFilters(variantInfoFilters);
+        query.setFields(List.of("\\open_access-1000Genomes\\data\\SYNTHETIC_AGE\\"));
+        query.setExpectedResultType(ResultType.DATAFRAME);
+
+        AsyncResult asyncResult = queryService.runQuery(query);
+
+        Thread.sleep(1000);
```
Comment (Luke-Sikina) [inline service/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/service/QueryServiceTest.java:52]: >:|

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/MultiValueQueryProcessor.java [hunk 2]
```diff
@@ -13,22 +13,21 @@
...
 
-    private Logger log = LoggerFactory.getLogger(PfbProcessor.class);
+    private Logger log = LoggerFactory.getLogger(MultiValueQueryProcessor.class);
 
 
     @Autowired
-    public PfbProcessor(AbstractProcessor abstractProcessor) {
+    public MultiValueQueryProcessor(AbstractProcessor abstractProcessor) {
         this.abstractProcessor = abstractProcessor;
-        ID_BATCH_SIZE = Integer.parseInt(System.getProperty("ID_BATCH_SIZE", "0"));
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/MultiValueQueryProcessor.java:31]: Undo this

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/MultiValueQueryProcessor.java [hunk 3]
```diff
@@ -13,22 +13,21 @@
...
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 @Component
-public class PfbProcessor implements HpdsProcessor {
+public class MultiValueQueryProcessor implements HpdsProcessor {
 
     public static final String PATIENT_ID_FIELD_NAME = "patient_id";
     private final int ID_BATCH_SIZE;
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/MultiValueQueryProcessor.java:21]: Value annotation

File: service/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/service/QueryServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+        System.out.println(asyncResult.getStatus());
```
Comment (ramari16) [inline service/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/service/QueryServiceTest.java:54]: Add validation

**Pull request merged**